concepticondata/references/references.bib

```diff
@@ -30,6 +30,17 @@ @article{Araujo1996
 	year = {1996},
 }
 
+@Article{Wang2021,
```

Comment:
BTW: I am happy to avoid unnecessary information. typically, we don't need publisher for articles, as we have the journal name. however, what I really encourage to look for is the volume of the journal and the issue number. If there's a DOI, no need to add an URL that is just the DOI.